### DISCUSSION TITLE
⚠️ Intel 2018 Mini HDMI / M1 Mini, MBP, Studio HDMI / M2 Mini HDMI not working / garbled screen when DDC is enabled

### DISCUSSION THREAD

**Discussion by Wyral** [MonitorControl/MonitorControl#1596] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

When changing the screen brightness and open things, the screen turns into black white beams.

### Steps to reproduce

1. Open MonitorControl
2. press wildly the shortcut for screen brightness
3. open settings, and browse a little bit through (while still pressing the brightness buttons) [1]
4. See the result [2]

[1] 
https://github.com/MonitorControl/MonitorControl/assets/104320525/95d649e8-97ae-456d-8ca2-a100a71e2c21

[2] 
https://github.com/MonitorControl/MonitorControl/assets/104320525/93d2dc77-461c-419f-a696-48696e4ecfe3
https://github.com/MonitorControl/MonitorControl/assets/104320525/bbe17a6b-7541-41d4-9232-162950d1d595
https://github.com/MonitorControl/MonitorControl/assets/104320525/1944a5e0-5460-4e84-8d09-d746b9641f63

### Expected behavior

Work with my Mac normally 

### Environment Information (please complete the following information)

```markdown
- macOS version: 14.0 (23A339)
- Mac model: Mac mini M1, 2020
- MonitorControl version: 4.2.0 Build 7048
- Monitor(s): 1
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment]
I assume you are using HDMI. MonitorControl does not support the M1 and entry level M2 Mini's HDMI port. You can however use [BetterDisplay](https://github.com/waydabber/BetterDisplay#readme). If you are using USB-C/DP then your display might not be DDC compatible.

**Comment by Wyral** [comment]
Yes, I am using HDMI.

**Comment by waydabber** [comment]
I'll just keep it here as a Known issue - although it's in the compatibility section but others will probably also search for this among issues.

**Comment by Addlai** [comment]
Same problem with Mac Mini 2020 M1 and LG 43in 4K Ultrafine over HDMI.

**Comment by waydabber** [comment]
@Addlai - please use [BetterDisplay](https://betterdisplay.pro) if you want DDC control over HDMI for these Macs. I do not plan to update MonitorControl with HDMI support. All DDC features are free in BetterDisplay.

**Comment by PhDLuffy** [comment]
> I assume you are using HDMI. MonitorControl does not support the M1 and entry level M2 Mini's HDMI port. You can however use [BetterDisplay](https://github.com/waydabber/BetterDisplay#readme). If you are using USB-C/DP then your display might not be DDC compatible.

Mac mini M1，MacOS 13.4.1，MC 4.2.0，AOC U2790B

mac hdmi to hdmi, volume bar work but sound no change

mac typec to hdmi, volume bar work but sound no change

mac typec to dp, volume bar work and sound control work perfactyly

i think for the hdmi, the ddc control signal may not come to the monitor,

the DP works perfactly.

**Comment by metehandemir** [comment]
I faced the same problem once with the Mac Mini 2018 i5.

**Comment by waydabber** [comment]
Hi @metehandemir - you can use [BetterDisplay](https://github.com/waydabber/BetterDisplay#readme) for HDMI support for the 2018 Mini (that is currently the only app that supports Intel 2018 HDMI DDC).
